core/src/main/res/color/filter_dialog_button_text.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="?attr/filter_dialog_text_active" android:state_checked="true" />
+    <item android:color="?attr/filter_dialog_text_inactive" />
```

Comment:
Instead of using new `attrs`, I think you can re-use `?attr/colorOnSecondary` (when active) and `?attr/textColorPrimary` (when not active). This saves a few additional definition and therefore ensures consistency.

Reply:
are you serious or are you joking at this point

Reply:
👍